frontend/src/metabase/admin/datamodel/components/revisions/RevisionDiff.jsx

```diff
@@ -42,7 +40,7 @@ export default class RevisionDiff extends Component {
           </div>
           <div>
             {this.props.property === "definition" ? (
-              <QueryDiff diff={this.props.diff} tableMetadata={tableMetadata} />
+              <QueryDiff diff={this.props.diff} />
```

Comment:
You can verify this by going to the `QueryDiff` - no `tableMetadata` is even mentioned there.